tools/perf/ResultsComparer/src/ResultsComparer/VsProfiler/Models/VsProfilerTypeAllocations.cs

```diff
@@ -0,0 +1,24 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="VsProfilerTypeAllocations.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using ResultsComparer.Core;
+
+namespace ResultsComparer.VsProfiler
+{
+    /// <summary>
+    /// Represents the type of entries in a
+    /// VS .NET Object Allocation Profiler Allocations report.

```

Comment:
"Allocation Profiler Allocations report" doesn't roll out smoothly from the tongue